docs/ems/privacy_notice.md

```diff
@@ -178,10 +179,18 @@ We track usage data for Element hosting services. When you are signed in to your
 
 Our analytics are powered by the Free and Open Source analytics platform [Matomo](https://matomo.org/), hosted entirely within our network. The servers which host this data are located in the EEA, specifically in London, United Kingdom. We don't share any analytics data with third parties.
 
-If you are using the Element chat app, you will be asked to opt-in to this anonymised data collection. Your use of the Service does not rely on your opt-in to this. When using any of the Element chat clients after opt-in, your visit to the service will be logged, alongside your device ID and agent. This allows us to understand usage patterns based on each platform. The mapping of this data is logged for 28 days - from this point on only aggregated data is kept, for operational and statistical purposes.
+If you are using the Element chat app, you will be asked to opt-in to this anonymised data collection. Your use of the Service does not rely on your opt-in to this. When using any of the Element chat clients after opt-in, your visit to the service will be logged, alongside your device ID and agent. This allows us to understand usage patterns based on each platform. The mapping of this data is logged for 28 days - from this point on only aggregated data is kept, for operational and statistical purposes. For further details on our collection and usage of analytics data, please see our [Cookie Policy](https://element.io/cookie-policy).
 
 When reporting errors we might collect some information to help us find a solution. This may include your IP address, hostname, full name and e-mail address. This information is collated in the application monitoring platform [Sentry](https://sentry.io/welcome/) and is subject to strict retention policies.
 
+**Location Information**
+
+We may collection location data on you, if you chose to use the static or live location sharing features within the Element app. This includes your longitude, altitude and latitude data in order to accurately calculate your precise location. 
+
+Location data is held within the room in which is shared in, so it will be encrypted in encrypted rooms and not encrypted in rooms where encryption is switched off. You will be shown a disclaimer during your first time using this feature, but please apply caution and consideration when sharing your personal data within the app.
+
+We use the third-party library [MapLibre](https://maplibre.org/) to provide you with map information.
```

Comment:
I don't think we need to mention MapLibre - this is just some code we use, similar to many other libraries.

Instead, we should mention MapTiler:

The Element clients use the third-party service [MapTiler](maptiler.com) to provide the images used to display maps.